<launch>
    <group ns="unav2">
        <rosparam command="load" file="$(find unav2_hardware)/config/control.yaml" />
        <node name="controller_spawner" pkg="controller_manager" type="spawner" args="joint_publisher velocity_controller" output="screen" />
    </group>
</launch>
